libs/wingc/src/type_check.rs

```diff
@@ -801,6 +801,8 @@ pub struct FunctionSignature {
 	/// - `$args$`: the arguments passed to this function call
 	/// - `$args_text$`: the original source text of the arguments passed to this function call, escaped
 	pub js_override: Option<String>,
+	/// During jsify, calls to this function will be replaced with this other function name
+	pub inflight_impl: Option<String>,
```

Comment:
I'd like to try avoiding this new machinery by reusing the `_toInflightType` methods we already support